app.py

```diff
@@ -201,9 +200,6 @@ def _request_end(resp):
 model_cache = get_model_cache(app.config)
 login_manager = LoginManager(app)
 prometheus = PrometheusPlugin(app)
```

Comment:
Is the idea to eventually move these other objects to singletons as well?

Reply:
the idea is to avoid all-in-one packages like `app`. Moving these objects into their own packages is the easiest way to achieve that.